packages/design-system/src/components/web-components/ds-tooltip/ds-tooltip.tsx

```diff
@@ -0,0 +1,120 @@
+import { define } from '../preactement/define';
+import { Tooltip, TooltipProps } from '../../Tooltip';
+import { Placement } from '@popperjs/core';
+import { parseBooleanAttr } from '../wrapperUtils';
+
+const attributes = [
+  'active-class-name',
+  'class-name',
+  'close-button-label',
+  'component',
+  'content-heading',
+  'dialog',
+  'interactive-border',
+  'inversed',
+  'max-width',
+  'offset',
+  'placement',
+  'root-id',
+  'show-close-button',
+  'title',
+  'transition-duration',
+  'trigger-aria-label',
+  'z-index',
+] as const;
+
+/* eslint-disable @typescript-eslint/no-namespace */
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-tooltip': React.DetailedHTMLProps<
+        React.HTMLAttributes<HTMLElement> & {
+          [K in (typeof attributes)[number]]?: string;
+        },
+        HTMLElement
+      >;
+    }
+  }
+}
+/* eslint-enable */
+
+interface WrapperProps
+  extends Omit<
+    TooltipProps,
+    | 'ariaLabel'
+    | 'contentHeading'
+    | 'dialog'
+    | 'inversed'
+    | 'showCloseButton'
+    | 'offset'
+    | 'placement'
+    | 'title'
+  > {
+  contentHeading?: string | TooltipProps['contentHeading'];
+  dialog?: string;
+  inversed?: string;
+  showCloseButton?: string;
+  placement?: string;
+  rootId?: string;
+  offset?: string;
+  title: string | TooltipProps['title'];
+}
+
+const parseOffset = (offset: string): TooltipProps['offset'] => {
+  if (typeof offset === 'string') {
+    const vals = offset.split(' ');
+    const [xOffset, yOffset] = vals;
+    return [parseInt(xOffset), parseInt(yOffset)];
+  }
+  // Else return the default offset value
+  return Tooltip.defaultProps.offset as TooltipProps['offset'];
+};
+
+const isPlacementValue = (location: string): location is Placement => {
+  return [
+    'auto',
+    'auto-start',
+    'auto-end',
+    'top',
+    'top-start',
+    'top-end',
+    'bottom',
+    'bottom-start',
+    'bottom-end',
+    'right',
+    'right-start',
+    'right-end',
+    'left',
+    'left-start',
+    'left-end',
+  ].includes(location);
+};
+
+const Wrapper = ({
+  contentHeading,
+  dialog,
+  inversed,
+  offset,
+  rootId,
+  title,
+  ...otherProps
+}: WrapperProps) => (
+  <Tooltip
+    {...otherProps}
+    ariaLabel={otherProps.triggerAriaLabel}
```

Comment:
OK, so I think the way this is setup actually handles all the cases we might be worried about re duplicative aria labelling. 
1. You can have both `aria-label` and `trigger-aria-label` exist on the component. The `trigger-aria-label` attribute will still apply to the child trigger element, and the `aria-label` value will apply only to the parent.
2. If you apply an `aria-label` and no `trigger-aria-label` attribute, the new `aria-label` attribute only exists on the parent web component container. It is not applied to the child trigger element.

As the code currently is, I think these are sufficiently distinct. Please let me know if you have any concerns @kim-cmsds @tamara-corbalt @pwolfert